### ISSUE TITLE
Theme Provider or Theme Service for v3

### ISSUE THREAD

**Issue by AathifMahir** [microsoft/fluentui-blazor#638]
Are there plans to develop Theme Provider or Theme Service for v3 or maybe v3.1, I guess it's going to be critical part of Fluent Theming System Going Forward

**Comment by vnbaaij** [comment]
No, there are no plans for that. Don't think it is a critical part as we 'only' follow the Fluent Design. Theming is already done in the components themselves. It is possible to tweak things to specific needs by means of the Design Token implementation. Examples of that can be found in the demo sites.

But interested to hear what others think though. Moving this to the discussion area as it more in its place there.
